### PR TITLE
Move the Redis chart to bitnami as update to 12.3.3

### PR DESCRIPTION
**Pull request by HazWard**
**What this PR does / Why we need it**:
The Redis chart was transitioned from stable to bitnami (https://github.com/helm/charts/issues/20969) as part of deprecation and the version was bumped to the latest available.

### CODE REVIEW

**Comment by aLekSer** [comment]
Thanks for your PR. New chart is needed to deploy OpenMatch. Previous one is not available:
https://www.docker.com/bitnami/redis

**Review by Laremere — APPROVED** [review]

**Comment by Laremere** [comment]
Fyi, I'm guessing you unchecked maintainers being allowed to add commits?  It means that I can't fix `This branch is out-of-date with the base branch`, so I can't merge it without you updating.  Up to you but thought I'd point that out for future PRs.

**Comment by HazWard** [comment]
I didn't there was a feature to let maintainers add commits, thanks! I bumped the version to the latest again since a new one came out.

**Pull request merged**
